Ccount.c

```diff
@@ -0,0 +1,19 @@
+/*Rebecca Hale, Assignment 2 Part 1*/
+#include <stdio.h>
+
+int main(){
+	int i;
+	char c;
```

Comment:
You shouldn't prompted the user to enter a string, which you then should have parsed to count the number of Cs.